models/users.model.js

```diff
@@ -1,5 +1,19 @@
 const db = require("../db/connection");
 
+/**
+ * TODO: consider potential refactor, selectUsers could accept parameters and
+ * manually build the query, as to not have both functions: selectUsers & selectUserByUsername
```

Comment:
You could look at this functionality if you think it is needed after core tasks. It would be one way to build a loose login function for front end.